python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -79,6 +84,38 @@ def status(args):
         ), f"not all components in conf file are online, check the previous output"
         print(f"all components in conf file are online")
 
+    if args.conn:
+        component_list = conn.execfetch("SHOW COMPONENTS", show=False)
+        t = PrettyTable()
+        t.title = "Connections"
+        t.field_names = ["Endpoint", "version", "cost_time", "extra"]
+        err = ""
+        for component in component_list:
+            try:
+                if component[1] == "taskmanager":
+                    response = requests.post(f"http://{component[0]}/openmldb.taskmanager.TaskManagerServer/GetVersion", json={}, timeout=1)
```

Comment:
timeouot=1 means 1 second? comment it

Reply:
add # the connection timeout is 1 second